man/nheko.1.adoc.in

```diff
@@ -0,0 +1,190 @@
+= nheko(1)
+:doctype:       manpage
+:Date:          2021-12-22
+:Revision:      @PROJECT_VERSION@
+:man source:    nheko
+:man manual:    General Commands Manual
+
+== NAME
+
+nheko - Desktop client for Matrix using Qt and C++17
+
+== SYNOPSIS
+
+*nheko* [_OPTIONS_]
+
+== DESCRIPTION
+
+The motivation behind the project is to provide a native desktop app for Matrix
+that feels more like a mainstream chat app (Element, Telegram etc) and less like
+an IRC client.
+
+== OPTIONS
+
+*-h*, *--help*::
+Displays help on commandline options.
+
+*--help-all*::
+Displays help including Qt specific options.
+
+*-v*, *--version*::
+Displays version information.
+
+*--debug*::
+Enables debug output.
+
+*-p*, *--profile*::
+Creates a unique profile, which allows you to log into several accounts at the
+same time and start multiple instances of nheko.
+
+== Keyboard shortcuts
+
+=== Room list
+
+*Ctrl-Up/Ctrl-Down*::
+Navigate within the room list.
+
+*Ctrl+k*::
```

Comment:
Why is this lowercase?

Reply:
Copied from <https://github.com/Nheko-Reborn/nheko/wiki/Shortcuts>. 😊 I can make them all uppercase?

Reply:
Well, if it is not too much effort, that would be nice, I think